### ISSUE TITLE
Is this javacpp-presets opencv supports WINE system.

### ISSUE THREAD

**Issue by vishalvaza** [bytedeco/javacpp-presets#1130]
I am trying to use javacpp-presets opencv with WINE operating system, Does it supports it?

**Comment by saudet** [comment]
What is WINE?

**Comment by matsjla** [comment]
> What is WINE?

See https://www.winehq.org/

I assume it might? run under wine, but because it's a windows-to-posix translator runtime you never know

**Comment by saudet** [comment]
Right, do you mean ReactOS maybe? It should work, yes, but I don't think it
was ever tested.

**Comment by bradh** [comment]
What are you trying to achieve with WINE? The presets should work fine on Linux and Windows, so why not just run it natively on whatever the OS is.
